Ruddarr/Views/Activity/ActivityView+Toolbar.swift

```diff
@@ -96,4 +99,20 @@ extension ActivityView {
                 .imageScale(.medium)
         }
     }
+    
+    var toolbarOpenSable: some View {
+        Link("Open Sable", destination: URL(string: sable) ?? Links.Sable)
```

Comment:
We should check if the URL can be opened first (to see if the app is installed). 

We also should display the button only for sabnzbs items, not all downdloads.

Reply:
Just made the modifications to the URL and the Open Sable link should now only show if it is installed. 

Not sure how you want to tackle the per-item scenario if it is in SABnzbd or not

Reply:
The payload for each queue item has download-client details, like its name/type.

Reply:
Got it. So in that case you want the button to be shown on the QueueItemSheet for items that are queued in SABnzbd?

Reply:
The most reliable field would probably be parsing `downloadId`.

>So in that case you want the button to be shown on the QueueItemSheet for items that are queued in SABnzbd?

Yep

Reply:
I'll work on that later this weekend. Going to a wedding today!

Reply:
```json
"downloadId": "SABnzbd_nzo_lef0l_33",
"protocol": "usenet",
"downloadClient": "SABnzbd",
```